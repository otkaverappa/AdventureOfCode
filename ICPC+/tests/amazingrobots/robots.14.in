20 20
####################
#X.................#
##################.#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#...................
#..................#
####################
2
19 19 3 N
18 19 3 N
15 20
####################
#..................#
...................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#...............####
#..................#
#.################.#
#.#..#...........#.#
#................#X#
####################
2
2 2 2 S
2 3 4 S
