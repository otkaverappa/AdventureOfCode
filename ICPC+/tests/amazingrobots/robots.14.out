68
N
N
N
W
W
W
N
N
N
N
N
N
N
N
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
N
W
E
E
E
E
E
E
E
E
E
E
E
E
E
E
E
E
E
S
S
W
S
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
E
